README.md

```diff
@@ -1,6 +1,37 @@
 # [Experimental] Strategic DDD Architecture Building Blocks
 
-This repository catalogues a language for describing architectural concepts from a Strategic Domain-Driven Design perspective.
+## Problem Space vs Solution Space: Example Scenario
 
-The goal of this repository is to create a shared language that practitioners can apply consistently across projects and domains. Less ambiguity should equally benefit newcomers to DDD and experienced practitioners.
+![Problem_vs_solution_space](problem_vs_solution_space.jpg)
 
+Book Company's mission is to improve every aspect of reading. They have noticed that many readers are frustrated when they forget to take books with them when they leave the house and can't read their books while commuting. 
+
+>Physical Books (subdomain) is part of problem space for Book Company
```

Comment:
This is the only phrase, I think is confusing. Strictly speaking, you are right of course but using **everywhere** DDD terms would be (for me at least) counterproductive. I would agree if it would be helpful but what cannot books just be books and nothing bound to DDD. It feels like it would transform a thing easy to understand in something that must be categorized (in a subdomain in this case). And opens the channel for other (not helpful) questions like "are you sure that books are subdomains and not domains?"

Reply:
Thanks for the feedback @yellowbrickc.

Can I infer from your comment that you are saying that there are no domains in the problem space? I.e. physical books is our problem space therefore it cannot be a (sub)domain?

On the topic of: 

"And opens the channel for other (not helpful) questions like "are you sure that books are subdomains and not domains?""

This unfortunately is the cost of fuzziness. We like terms like domain and subdomain because they are fuzzy and can be applied in new and novel scenarios. But as a trade-off, it's hard to pin down precisely what they mean.

And just to clarify, with this diagram, I am just trying to visualise how I think each part of the system would be labelled according to DDD terminology. In DDD, the subdomain (physical books) are part of the problem space. I wouldn't use this diagram to teach DDD because I don't think problem/solution space are clear enough to be uuseful.

If you get chance, I would love to see you could create a version of this diagram labelling each part how you interpret DDD. Then we can see where we have common ground and where we interpret parts of DDD differently (open invitation to everyone else as well).

Reply:
@NTCoding 

>  there are no domains in the problem space

Just like Mathias says "what's a solution space for one, is the problem space for another" I just don't think that it helps by the clarification. I see it like @skleanthous, I would try to leave out everything creating even more fuzziness. I do not think, that we should see everything in DDD terms. These terms are not originated in DDD, they just used a lot by people in DDD (IMO).

I'm waiting (eagerly) for the end of the year and my holiday (meaning not working) and I hope that I will find the energy to create a fitting diagram but as I said: I agree with the rest, I think it describes well the situations.